Tasks/JenkinsDownloadArtifacts/ArtifactDetails/JenkinsRestClient.ts

```diff
@@ -0,0 +1,145 @@
+import * as Q from 'q';
+import * as tl from 'vsts-task-lib/task';
+
+var handlebars = require('handlebars');
+var request = require('request');
+
+export class JenkinsRestClient {
+    constructor() {
+        this.RegisterCustomerHandleBars();
+    }
+
+    public RegisterCustomerHandleBars(): void {
+        handlebars.registerHelper('CaseIgnoreEqual', function(lhs, rhs, options) {
```

Comment:
Should this be "caseIgnoreEqual" to match the other helper names?

Reply:
Fixed.